test/cli/test_helper.rb

```diff
@@ -0,0 +1,3 @@
+require "aruba/api"
+require "minitest/autorun"
```

Comment:
I *think* you can drop this line, since `minitest/autorun` is required transitively though `helpers`.

Reply:
y. i was a little hesitant to tighten the cli tests' dependence on the unit tests' `helpers`, but if you are ok with doing so, i can remove the extra require for `minitest/autorun`. 

in the other direction, if you were inclined to add `require "aruba/api"` to `helpers`, we could ditch `cli/test_helpers.rb altogether.

Reply:
> in the other direction, if you were inclined to add require "aruba/api" to helpers, we could ditch cli/test_helpers.rb altogether.

That sounds good to me - one helper file is better than two.

Reply:
...i guess as a third option, we could just bring what we actually need from `helpers` to `cli/test_helpers` and enforce independence between the two suites

(btw, aruba's docs reference and the provided initializer creates a `test_helper.rb` file, thus the name difference with the unit tests)

Reply:
> That sounds good to me - one helper file is better than two.

oops. missed your reply in the back-and-forth. we'll go with option two then

Reply:
> oops. missed your reply in the back-and-forth. we'll go with option two then

Yep, sounds good.